plugins/webp-uploads/picture-element.php

```diff
@@ -56,7 +56,7 @@ function webp_uploads_wrap_image_in_picture( string $image, string $context, int
 	/**
 	 * Filter the image mime types that can be used for the <picture> element.
 	 *
-	 * Default is: ['image/avif', 'image/webp']. Returning an empty array will skip using the `picture` element.
+	 * Default is: ['image/avif', 'image/webp', 'image/jpeg', 'image/png]. Returning an empty array will skip using the `picture` element.
```

Comment:
```suggestion
	 * Default is: ['image/avif', 'image/webp']. Returning an empty array will skip using the `picture` element.
```

In Picture > source we only needs additional mime not the fallback images. In last `2.1.0` version we remove 'image/jpeg' from the list.

Reply:
Ah, right. Thanks